### ISSUE TITLE
Internal error 500 with lazy load

### ISSUE THREAD

**Issue by lucl22** [jonnnnyw/php-phantomjs#107]
Hi,

when I test with http://jonnyw.kiwi/tests/test-lazy.php, everything is ok (got the page and the 'loaded' content). I have an internal error after several minutes with a website I'm scrapping (but no error on 'basic usage script' with delay).

Here is my code:

`
    require 'vendor/autoload.php';
    use JonnyW\PhantomJs\Client;

    $url = "http://jonnyw.kiwi/tests/test-lazy.php";

       $client = Client::getInstance();
       $client->isLazy();

        $request  = $client->getMessageFactory()->createRequest();
        $request->setMethod('GET');
        $request->setUrl($url);
        //$request->setTimeout(5000);

        $response = $client->getMessageFactory()->createResponse();
        $client->send($request, $response);

        echo $response->getContent();`

Thanks for your help.

ps: sorry for my english, it's not my primary language.

**Comment by jonnnnyw** [comment]
Would you be able to provide me with the URL of the website you are scraping so I can test it myself?

**Comment by lucl22** [comment]
Sorry, I forgot the link.

Url = http://www.canadiantire.ca/fr/pdp/dewalt-20v-max-li-ion-drill-impact-driver-kit-0543177p.html

Best regards.

**Comment by jonnnnyw** [comment]
I managed to find out what the issue was. The lazy script was logging loaded resources to the debug output but this was causing the script to hang when their were are large number of resources. I have removed resource logging and it successfully loads the URL you provided.

I have released `v4.5.1` with the fix.

You will want to clear the compiler cache before trying again:

```
$client = Client::getInstance();
$client->getProcedureCompiler()->clearCache();
```

Remember to remove the `$client->getProcedureCompiler()->clearCache()` from your script after you've cleared the cache as this will slow down execution massively.

Let me know how you get on.
